persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/DsePersistence.java

```diff
@@ -462,7 +463,16 @@ public boolean isInSchemaAgreement() {
     // Also note that in theory getSchemaVersion can return null for some nodes, and if it does
     // the code below will likely return false (the null will be an element on its own), but that's
     // probably the right answer in that case. In practice, this shouldn't be a problem though.
-    return Gossiper.instance.getLiveTokenOwners().stream()
+
+    // Important: This must include all nodes including fat clients, otherwise we'll get write
+    // errors
```

Comment:
ugh, why does it format like this?

Reply:
over line length limit?

Reply:
try merging with the next line?

Reply:
It was, but the automatic formatter did this.